src/Routers/FunctionsRouter.js

```diff
@@ -140,7 +140,7 @@ export class FunctionsRouter extends PromiseRouter {
         result => {
           try {
             const cleanResult = logger.truncateLogMessage(JSON.stringify(result.response.result));
-            logger.info(
+            logger[req.config.logLevels.cloudFunctionRan](
```

Comment:
Can you really use `req.config.logLevels.cloudFunctionRan` or could there be cases where `req.config.logLevels` is undefined?

Reply:
It seems that the default value is always applied, so no, there(s no case where ```req.config.logLevels``` is ```undefined```. I updated unit test to validate this.